package.json

```
@@ -2,11 +2,14 @@
   "name": "sd-0x-project-cookmaster-v2",
   "version": "1.0.0",
   "description": "Você já usa o GitHub diariamente para desenvolver os exercícios, certo? Agora, para desenvolver os projetos, você deverá seguir as instruções a seguir. Fique atento a cada passo, e se tiver qualquer dúvida, nos envie por Slack! #vqv 🚀",
-  "main": "index.js",
+  "main": "./src/api/server.js",
   "scripts": {
-    "test": "jest --runInBand  --detectOpenHandles --forceExit",
-    "start": "node index.js",
-    "debug": "nodemon index.js",
+    "test": "jest --runInBand --detectOpenHandles --forceExit",
+    "start": "node .",
+    "dev": "nodemon .",
```

Não seria assim @mjgargani  ? Ou apenas com o '.' funciona ?

```suggestion
    "start": "node ./src/api/server.js",
    "dev": "nodemon ./src/api/server.js",
```

---

@vinicius-vasconcelos  Não há necessidade de passar o caminho no script, uma vez que ele está definido no parâmetro [`main`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#main) do `package.json`, o node identifica e roda o entry point `main` por padrão.